examples/classification/requirements.txt

```diff
@@ -0,0 +1,12 @@
+requests==2.25.1
+opencv_python==4.5.3.56
```

Comment:
nit: let's delete this if possible (see comments in helper.py)

Reply:
🆒